_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -206,6 +206,95 @@ You will most likely not need to specify any parameters except for `location`. F
 
 You will most likely not need to specify any parameters except for `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/).
 
+
+### Azure storage account
+
+These steps explain how to register a snapshot repository backed by an Azure storage account for an OpenSearch cluster deployed using Helm.
+
+1. Create an Azure storage account. Then create a container within the storage account.
+
+1. Create a Docker image with the `repository-azure` plugin.
+
+   create-keystore.yaml
+
+   ```bash
+   #!/bin/bash
+
+   /usr/share/opensearch/bin/opensearch-keystore create
+   echo $AZURE_SNAPSHOT_STORAGE_ACCOUNT | /usr/share/opensearch/bin/opensearch-keystore add --stdin azure.client.default.account
+   echo $AZURE_SNAPSHOT_STORAGE_ACCOUNT_KEY | /usr/share/opensearch/bin/opensearch-keystore add --stdin azure.client.default.key
+   cp /usr/share/opensearch/config/opensearch.keystore /tmp/keystore/opensearch.keystore
+   ```
+
+   Dockerfile
+
+   ```docker
+   FROM opensearchproject/opensearch:{{site.opensearch_version}}
+
+   RUN /usr/share/opensearch/bin/opensearch-plugin install --batch repository-azure
+   COPY --chmod=0775 create-keystore.sh create-keystore.sh
+   ```
+
+   ```
+   docker build -t opensearch-custom:{{site.opensearch_version}} -f Dockerfile .
+   ```
+
+
+1. Create a Kubernetes secret containing the Azure storage account key
```

Comment:
```suggestion
1. Create a Kubernetes secret containing the Azure storage account key using the following command:

```

Reply:
I have only added the Kubernetes manifest here. Shall add the `kubectl` command as well? And then say `Create a Kubernetes secret containing the Azure storage account key using the following manifest and command:`

Reply:
That works for me @nilushancosta